unittests/test_cli.py

```diff
@@ -234,7 +234,7 @@ def test_failure_stats(self):
         returncode, stdout, stderr = self._run_reframe()
 
         assert r'FAILURE STATISTICS' in stdout
-        assert r'sanity      1     [SanityFailureCheck' in stdout
+        assert r'sanity        1     [SanityFailureCheck' in stdout
```

Comment:
I don't understand this change. This should be from another PR.

Reply:
Me neither. I have merged from master